src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.task.Task;
+
+/**
+ * Adds an event task to the task book.
+ */
+public abstract class AddTaskCommand extends Command {
+
```

Comment:
What's the point of introducing this base class? I don't see the use of LSP anywhere.